tests/models/bart/test_modeling_tf_bart.py

```diff
@@ -293,8 +293,8 @@ def _get_word_embedding_weight(model, embedding_layer):
                                 models_equal = False
                     self.assertTrue(models_equal)
 
+    @unittest.skip(reason="Test takes too long to run (>30sec) causing the CI to fail.")
```

Comment:
We also have a `@tooslow` decorator for this case (also applicable to other tests with the same skip reason)

https://github.com/huggingface/transformers/blob/ccc089780415445768bcfd3ac4418cec20353484/src/transformers/testing_utils.py#L219

Reply:
Didn't know that - thanks for pointing out! I've updated the tests